File: front/src/models/Attendance.ts
import type {EventDate} from "@/models/EventDate.ts";
import type {Attendee} from "@/models/Attendee.ts";

export class Attendance {
  public date : EventDate
  public attendee : Attendee

  public constructor(date : EventDate, attendee : Attendee) {
    this.date = date;
    this.attendee = attendee;
  }
}

File: back/src/main/java/be/naaturel/letsmeet/core/models/Attendance.java
package be.naaturel.letsmeet.core.models;

public class Attendance {

    private EventDate date;
    private Attendee attendee;

    public Attendance(EventDate date, Attendee attendee) {
        this.date = date;
        this.attendee = attendee;
    }

    public Attendee getAttendee() {
        return attendee;
    }

    public EventDate getDate() {
        return date;
    }
}
